ocaml/idl/datamodel_types.ml

```diff
@@ -56,6 +56,7 @@ let rel_indigo = "indigo"
 let rel_dundee = "dundee"
 let rel_dundee_plus = "dundee-plus"
 let rel_ely = "ely"
+let rel_falcon = "falcon"
```

Comment:
Please would you split out this file's changes into a separate commit?
Defining `rel_falcon` and its place in `release_order` is not specific to the new Pool funciton.